src/feautures/MovieList/styled.js

```diff
@@ -3,6 +3,15 @@ import styled from "styled-components";
 export const StyledMovies = styled.div`
     display: grid;
     grid-template-columns: auto auto auto auto;
+    justify-content: space-evenly;
     grid-gap: 24px;
-    margin-bottom: 40px;
-    `;
\ No newline at end of file
+    margin: 0 auto 40px auto;
+
+    @media (max-width: 632px) {
+        grid-template-columns: auto;
+    };
+
+    @media (min-width: 633px) and (max-width: ${({ theme }) => theme.breakpoint.mobile}px) {
+        grid-template-columns: auto auto;
+    };
+`;
```

Comment:
I think one more media query is required:
![obraz](https://user-images.githubusercontent.com/118211320/220996870-c97c6af5-072e-4f69-9bfa-25e5424f74f3.png)

Reply:
Fixed :)